src/test/java/io/jenkins/plugins/opentelemetry/BaseIntegrationTest.java

```diff
@@ -101,6 +104,15 @@ public static void beforeClass() throws Exception {
         jenkinsRule.waitUntilNoActivity();
         LOGGER.log(Level.INFO, "Jenkins started");
 
+        // Update all sites to reload available plugins.
+        jenkinsRule.jenkins.getUpdateCenter().updateAllSites();
```

Comment:
Why? This is almost never something you would want to do in a `JenkinsRule` test. The plugins used by the test will be those in the test classpath, and if you want to test against newer plugins (or core), update your POM (normally bumping the `bom` version). https://groups.google.com/g/jenkinsci-dev/c/WeLLd7cJrCo/m/17LmvlT4AAAJ

Reply:
I haven't used the JenkinsRule somewhere other than this patch and I'm not that familiar with it. I can see why updating the sites might not be right for a test. But it doesn't seem to me that's the root cause of the issue on the google group discussion.

Reply:
I removed that code recently at https://github.com/jenkinsci/opentelemetry-plugin/pull/999 but somehow, I made the PR against the wrong target, the code is not needed.

Reply:
I just remove it on main https://github.com/jenkinsci/opentelemetry-plugin/commit/e1da710f9097b1d6f4c242bd498cb8a061404aab